packages/grid/src/vaadin-grid-column.js

```diff
@@ -46,6 +46,20 @@ export const ColumnBaseMixin = (superClass) =>
           value: false,
         },
 
+        /**
+         * When true, then in the shadow DOM, the column cells will be rendered with the `role="rowheader"` attribute.
+         * - This will cause the column will be handled as a rowHeader and read properly by screen readers.
+         *
+         * When false, then in the shadow DOM, the column cells will be rendered with the `role="gridcell"` attribute.
```

Comment:
```suggestion
        /**
         * When true, the cells for this column will be rendered with the `role` attribute
         * set as `rowheader`, instead of the `gridcell` role value used by default.
         *
         * When a column is set as row header, its cells will be announced by screen readers
         * while navigating to help user identify the current row as uniquely as possible.
```